yabt/buildcontext.py

```diff
@@ -287,13 +287,16 @@ def fail_notifier(ex):
                 # "failed to run" errors.
                 # see: https://github.com/resonai/ybt/issues/124
                 try:
-                    if isinstance(ex, CalledProcessError) and ex.stdout:
+                    if isinstance(ex, CalledProcessError) and \
+                       (ex.stdout or ex.stderr):
                         # TODO(Dana) When ex.stdout has non ascii chars the
                         # call to sys.stdout.write crashes in the inner
                         # function, when colorama/ansitowin32.py assumes that
                         # the text is ascii encoded.
-                        sys.stdout.write(ex.stdout.decode('utf-8'))
-                        sys.stderr.write(ex.stderr.decode('utf-8'))
+                        if ex.stdout:
```

Comment:
<img width="906" alt="image" src="https://github.com/resonai/ybt/assets/38324390/b79cd783-fcf6-43e0-81b5-23e3f780af81">

you can run `py.test --flake8 --cov=yabt --with-slow` to run tests locally (probably you can drop the `--with-slow` to just test lint)